src/app/features/change-history/change-history-dialog/change-history-dialog.component.scss

```
@@ -0,0 +1,96 @@
+
+.empty-state {
+  padding: sizes.$large sizes.$regular;
+  text-align: center;
+}
+
+// card-style expansion panels: each panel is a separated, elevated card with a
+// border and spacing between them; the expanded header gets a light-blue tint
+.timeline {
+  display: block;
+
+  ::ng-deep .mat-expansion-panel {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)